Noita-Community/streamer-wands-backend
# Add 1000th wand sprite

Testing method:
1. Use [seed changer](https://steamcommunity.com/sharedfiles/filedetails/?id=2284931352) and maybe cheat gui to get ASE+tele wand
2. in the mod settings set the seed to `1006815449`
3. pick up the wand ![image](https://github.com/myndzi/streamer-wands-backend/assets/106106310/1611c79b-a82d-4e40-99ce-05ea31c5cd03)
4. Check if Streamer Wands correctly show the sprite.
